scaling_performance/scaling_cluster_monitoring.adoc

```diff
@@ -21,16 +21,17 @@ and components metrics in one dashboard interface, Grafana.
 
 This topic provides information on scaling the cluster monitoring operator.
 
+If you want to use Prometheus with persistent storage, you must set the following variable in
```

Comment:
you must set the `openshift_cluster_monitoring_operator_prometheus_storage_enabled=true` in your Ansible inventory file to `true`.